keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -586,8 +589,12 @@ feature cannot be used. Pods using the new `VolumeSource` combined with a not
 supported container runtime version will fail to run on the node, because the
 `Mount.host_path` field is not set for those mounts.
 
-For security reasons, volume mounts should set the [`noexec`] and `ro`
-(read-only) options by default.
+For the most conservative security approach, image volumes should be mounted by
```

Comment:
I think it could be clearer that this behavior is ultimately up to the runtime as it's not toggle-able by the readOnly field in the k8s mount object. right now the phrasing is pretty passive.

Reply:
I added another phrase to outline that it's up to the runtime :+1: